thumbor/transformer.py

```diff
@@ -340,6 +340,12 @@ def fit_in_resize(self):
             resize_height = self.target_height
             resize_width = round(source_width * self.target_height / source_height)
 
+        # ensure that filter should work on the real image size and not on the request
+        # size which might be smaller than the resized image in case `full-fit-in` is
+        # being used
+        self.context.request.width = int(max(self.context.request.width, resize_width))
```

Comment:
This is throwing exception in case when width or height specified as 'orig'
`/unsafe/fit-in/200xorig/http://www.wallsave.com/wallpapers/2560x1600/cloud/66266/cloud-zeom-computing-g-free-66266.jpg`
that would return unresized image